Cargo.toml

```diff
@@ -13,7 +13,11 @@ skrifa = "0.15.1"
 wasm-bindgen = "0.2"
 js-sys = "0.3.65"
 woff2 = "0.3.0"
+iconimation = { path = "../iconimation" }
+read-fonts = "0.15.1"
 
 [patch.crates-io]
 skrifa = { git="https://github.com/googlefonts/fontations.git", branch="main" }
+read-fonts = { git="https://github.com/googlefonts/fontations.git", branch="main" }
+write-fonts = { git="https://github.com/googlefonts/fontations.git", branch="main" }
```

Comment:
We can get read through skrifa. Also we prefer versioned deps over git deps and git deps over path deps so we should make iconimation a git dep and write-fonts and skia into versioned deps. I'll fix that post-merge.